app/models/user_video.rb

```diff
@@ -1,4 +1,8 @@
 class UserVideo < ApplicationRecord
   belongs_to :video
   belongs_to :user
+
+  def self.bookmarked_vidoes
+    Video.joins(%I[tutorial user_videos]).order(:tutorial_id, :position)
```

Comment:
This looks good! I wonder if this will return all videos or just the UserVideos

Reply:
I think it will just be bookmarked videos because it joins user_videos... but probably a test we can add.  Good call.